src/components/migrationToV3.stories.mdx

```diff
@@ -63,17 +66,18 @@ const MyApp = () => {
     const data = await response.json();
     const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
     const eventController = new DefaultEventController(
-        meetingSessionConfiguration, 
-        meetingManager.createLogger(meetingSessionConfiguration), 
-        new NoOpEventReporter()
-      );
-    const joinData = {
-      meetingSessionConfiguration: meetingSessionConfiguration,
-      eventController: eventController
-    };
+      meetingSessionConfiguration, 
+      meetingManager.createLogger(meetingSessionConfiguration), 
+      new NoOpEventReporter()
+    );
 
     try {
-      await meetingManager.join(joinData);
+      await meetingManager.join(
+        meetingSessionConfiguration,
+        options:  {
+          eventController: eventController,
+        }
```

Comment:
This is not right, the `options` is not a named parameter.

Suggest you to declare each parameter first, and then use them.
For example:

```js
const config = new MeetingSessionConfiguration(...);
const options = new MeetingManagerJoinOptions(...);
await meetingManager.join(config, options);
```

Reply:
Wait there is no MeetingManagerJoinOptions class, its just a type right?

Reply:
Oh yes that's right. Edited.